### PR TITLE
Make instant_article_id a BigInteger

### PR DESCRIPTION
**Pull request by MichaelButkovic**

### CODE REVIEW

File: bulbs/videos/migrations/0002_auto_20160613_1116.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```
Comment (MichaelButkovic) [inline bulbs/videos/migrations/0002_auto_20160613_1116.py]: @mparent61 This file was generated when I generated my migrations. Do we want to add this?

**Comment by MichaelButkovic** [comment]
@mparent61 @spra85 @benghaziboy related sentry error http://sentry.onion.com/onion/onion-test/issues/20461/

**Comment by mparent61** [comment]
👍

File: bulbs/videos/migrations/0002_auto_20160613_1116.py (hunk 1 above)
Comment (mparent61) [inline bulbs/videos/migrations/0002_auto_20160613_1116.py]: Ah, I guess this new migration is needed after I reverted to the default ID behavior. Sure, let's add it

**Pull request merged**